### DISCUSSION TITLE
Push Notifications

### DISCUSSION THREAD

**Discussion by tusharbhambere** [serverpod/serverpod#414] [Ideas]
How will can send push Notifications or Trigger One Signal/Firebase FCM Notification in Serverpod

**Comment by yahu1031** [comment]
I don't think we have such a package or plugin in serverpod. Incase if it doesn't exist, and if you are interested in developing one, please, you are welcome to do it and raise a PR to this repo. 😇

**Comment by tusharbhambere** [comment]
in Flutter for Notifications there are Most Commen 2 Options Onesignal and FCM(Firebase) 

For One Signal they have are own Api we just Need to call after some action perform. we can easily store app_id in Our Database 

For FCM I thing when Serverpod start supporting Cloud Function or Firebase Make SDK for Serverpod  It's Very Handy to use All Firebase Features

**Comment by vlidholt** [comment]
This should be added to future versions for sure!

I added a trackable issue here: https://github.com/serverpod/serverpod/issues/439
